### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mmghannam**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```
Comment (Joao-Dionisio) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]: Is this necessary, Mo?

**Review by Joao-Dionisio — COMMENTED** [review]
Added a comment.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (Joao-Dionisio) [reply .github/ISSUE_TEMPLATE/feature_request.md:16]: The alternatives you've considered, I mean.

Comment (mmghannam) [reply .github/ISSUE_TEMPLATE/feature_request.md:16]: Yeah maybe it's too much

**Review by Joao-Dionisio — APPROVED** [review]

**Pull request merged**